### DISCUSSION TITLE
Monthly ESMValTool community meeting August 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#41] [Meetings]
The next monthly ESMValTool community meeting will be in August. The result of the poll is that the meeting will take place on

Friday, August 5 at 14:00 CEST (13:00 BST).

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

@ESMValGroup/esmvaltool-developmentteam

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we can decide to talk about specific topics with the entire group or make smaller breakout groups to continue the discussion there.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/33#discussioncomment-3121324

**Comment by ehogan** [comment]
Quick question: did a calendar invite get sent for this meeting?

**Reply by bouweandela** [reply]
Yes, did you not receive it?

**Reply by ehogan** [reply]
No :( I definitely got the July invite, but not the August one.

**Reply by valeriupredoi** [reply]
MO email being unreliable? No way that's possible :laughing:

**Reply by ehogan** [reply]
Hehe, yeah, I was going to check that next 😛

**Reply by valeriupredoi** [reply]
https://github.com/ESMValGroup/Community/discussions/41

**Reply by valeriupredoi** [reply]
ah well, here's a first, I added a link to this very post :rofl:

**Reply by bouweandela** [reply]
You can see the message in the mailing list archive here: https://www.listserv.dfn.de/sympa/arc/esmvaltool/2022-08/

**Reply by ehogan** [reply]
The message wasn't trapped by our quarantine, so I'm going to put it down to a glitch in the matrix 😝

**Reply by ehogan** [reply]
Hmmm, so the invite just came in 🤔

**Comment by ehogan** [comment]
Would it be possible to discuss https://github.com/ESMValGroup/ESMValTool/discussions/2723 at this meeting, please?

**Reply by bouweandela** [reply]
@ehogan Asked at the meeting if there are any objections to the proposal. So please share any ideas you have with her. She will start implementing this next week and do a demo at the next monthly meeting.

**Comment by ehogan** [comment]
Would it be possible for @mo-tgeddes to share the dates for the autumn workshop at this meeting, please? See https://github.com/ESMValGroup/Community/discussions/45 :)

**Comment by ehogan** [comment]
Would there be time for @mo-tgeddes to demo the CLA assistant at this meeting? :)

**Reply by ehogan** [reply]
It should only take ~5 minutes :)

**Reply by bouweandela** [reply]
@mo-tgeddes Showed a demo of the CLA assistant using this repository: https://github.com/mo-tgeddes/demo1/. It can be seen in action on the pull requests, e.g. https://github.com/mo-tgeddes/demo1/pull/16.

**Reply by bouweandela** [reply]
@axel-lauer Asked where the information on who signed it is stored

**Reply by bouweandela** [reply]
@rswamina Asked if there is an issue open about this already. @ehogan said not yet.

**Reply by ehogan** [reply]
@rswamina also asked whether the CLA would need re-signing regularly e.g. every year :)

**Reply by mo-tgeddes** [reply]
> @axel-lauer Asked where the information on who signed it is stored

The data is stored in a a cosmosDB (mongoDB compatible) hosted on Microsoft Azure in Europe.
[Here](https://github.com/cla-assistant/cla-assistant#where-is-the-list-of-signees-stored) is where I found the information if you want to know more. : )

**Reply by mo-tgeddes** [reply]
> @rswamina also asked whether the CLA would need re-signing regularly e.g. every year :)

Apologies for the delay. The CLA won't time out, you will only have to resign it if there are changes made to it. You can withdraw your signature through the CLA user interface if you need to.

**Comment by valeriupredoi** [comment]
gonna also ask who is about in August so we can get a Tech Lead call meeting too - if it's too few of us in the trenches and not on some beaches then I'll send around a Doodle for the TL call :+1:

**Comment by bouweandela** [comment]
Update round summary:

- @ehogan Would like to talk about the topics before (recipe test workflow, workshop at met office, demo cla assistant)
- @rswamina Working on extracting IPCC AR6 regions, getting feedback on tutorials
- @axel-lauer Working on support for new datasets. Please have a look at the pull requests for updating the documentation on clarifying the @ESMValGroup/esmvaltool-recipe-maintainers role in https://github.com/ESMValGroup/ESMValTool/pull/2719 and the new policy for broken recipes in https://github.com/ESMValGroup/ESMValTool/pull/2720
- @tepmo42: developing jetstream recipe
- @bsolino: working on integrating analysis into ESMValTool
- @catherinehardacre: working on support for new datasets
- @glpotter: working on CMORizer, would like some help with the YAML
- @rswamina mentioned that @remi-kazeroni Is updating the tutorial regarding CMORizers
- @bouweandela is working on adding an `esmvalcore.dataset.Dataset` object that can be used to load datasets in Python without using a recipe in https://github.com/ESMValGroup/ESMValCore/pull/1609. By using this object also internally to load recipes, the maintainability of the recipe parsing code can be improved and several open issues can be solved more easily.

**Comment by bouweandela** [comment]
CMORizers
- @glpotter is having trouble making the CMORizer script find his updated MERRA2.yml file (https://github.com/ESMValGroup/ESMValTool/issues/2718).
- @bouweandela  will send @glpotter the contact info of our PIs @hb326 and @alistairsellar 

@catherinehardacre asked about CMORizing station data. Previous discussion on the topic was here ESMValGroup/ESMValTool#496 https://github.com/ESMValGroup/ESMValTool/issues/1119. The recommendation was to put all the data in a single file. The CF-Conventions for storing station timesries data are available [here](https://cfconventions.org/cf-conventions/cf-conventions.html#time-series-data), other types are also available. Documentation for creating a new CMORizer lives [here](https://docs.esmvaltool.org/en/latest/community/dataset.html).

**Comment by bouweandela** [comment]
@valeriupredoi Will organize the next @ESMValGroup/technical-lead-development-team meeting end of August

**Reply by valeriupredoi** [reply]
yep - on it, even it's well into September now :laughing:

**Comment by glpotter** [comment]
Thanks,I just did the pip install -e .[development] and it works now. I’ll go ahead an add the 2D and 3D variables into my MERRA2.yml.

 Jerry
